module.rb

```diff
@@ -0,0 +1,14 @@
+require 'csv'
+
+module Read_CSV_File
```

Comment:
this module is not only reading but also writing. 
A better name would be `FileUtility` or `FileUtilities` 
If you need to save to xml next time. Is this module reusable?